ViewModel/Adminhtml/Common.php

```diff
@@ -217,6 +229,44 @@ public function isClickAnalyticsTurnedOnInAdmin()
         return $this->configHelper->isClickConversionAnalyticsEnabled();
     }
 
+    public function isMysqlUsed()
+    {
+        $isMysql = false;
+        if ($this->configHelper->getBackendSearchEngine() === 'mysql') {
+            $isMysql = true;
+        }
+
+        return $isMysql;
```

Comment:
Would the following work?
```return $this->configHelper->getBackendSearchEngine() === 'mysql';```

Reply:
Yes totally, sorry the function was more complex at the beginning and I forgot to simplfy it /o\ .